daraja/src/main/java/com/github/daraja/utils/Utils.kt

```diff
@@ -39,3 +46,44 @@ fun getPassword(businessShortCode: String, passkey: String, timestamp: String):
     // encode the password to Base64
     return Base64.encodeToString(str.toByteArray(), Base64.NO_WRAP)
 }
+
+suspend fun <T> safeApiCall(
```

Comment:
It would be good if we change the visibility of this function to internal since a dev using the lib doesn't interact with it.

Reply:
Noted